.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+# Jira link(s)
```

Comment:
Bearing in mind that this is a public repository, we shouldn't be adding links to Jiras in PRs as only Red Hat associates would have access to them